int8 steer_offset_inc   # one of {-1, 0, 1}
